### PR TITLE
[Bifrost] Fixing overload, among other things

### PR DESCRIPTION
**Pull request by AhmedSoliman**
Adds a whole lot of logging temporarily as well.

### CODE REVIEW

File: crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs [hunk 1]
```diff
@@ -288,13 +304,26 @@ impl<T: TransportConnect> SequencerAppender<T> {
...
                 Ok(Some(result)) => result,
-                Ok(None) => break, //no more tasks
-                Err(_) => {
+                Ok(None) => break, // no more tasks
+                Err(elapsed) => {
+                    // if we have already ackowledged this append, it's okay to retire.
+                    if self.commit_resolver.is_none() {
+                        tracing::debug!(%pending_servers, %wave, ?spread, ?elapsed, responses=?checker, "Some servers didn't store this batch, but append was committed, giving up");
+                        return SequencerAppenderState::Done;
+                    }
```
Comment (muhamadazmy) [inline crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs:313]: Nice

File: crates/bifrost/src/providers/replicated_loglet/network.rs [hunk 2]
```diff
@@ -172,33 +172,15 @@ impl RequestPump {
...
-                    header: CommonResponseHeader {
-                        known_global_tail: Some(tail.offset()),
-                        sealed: Some(tail.is_sealed()),
-                        status: SequencerStatus::Ok,
-                    },
-                };
-                let _ = reciprocal.prepare(sequencer_state).try_send();
-            }
-            Err(err) => {
-                // Well, we couldn't finish FindTail. Very likely we failed in CheckSeal. In all
```
Comment (tillrohrmann) [inline crates/bifrost/src/providers/replicated_loglet/network.rs:187]: CheckSeal happens periodically, right?

Comment (AhmedSoliman) [reply crates/bifrost/src/providers/replicated_loglet/network.rs:187]: Yes. It's run by a periodic find tail, albeit this solution is still a shortcut and will likely cause "restatectl logs reconfigure" to crash. But that is not a blocker at the moment.

Comment (tillrohrmann) [reply crates/bifrost/src/providers/replicated_loglet/network.rs:187]: What's the problem for `restatectl logs reconfigure` now?

**Pull request merged**